### PR TITLE
Natalia/stone microservice

### PR DESCRIPTION
**Pull request by Isaribi**
Adding kamyk CQRS

### CODE REVIEW

File: server/libraries/commons/commons-stone/pom.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <lombok.version>1.18.34</lombok.version>
+    </properties>
+
+    <dependencies>
+
```
Comment (Saszq) [inline server/libraries/commons/commons-stone/pom.xml:22]: Whitespace?

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/StoneKafkaConstants.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.kamyczki.commons.stone;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class StoneKafkaConstants {
+
```
Comment (Saszq) [inline server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/StoneKafkaConstants.java:7]: whitespace here too, dont think intended

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneCreatedEventDto.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import lombok.*;
+
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString(callSuper = true)
+public class StoneCreatedEventDto extends StoneEventDto{
+
```
Comment (Saszq) [inline server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneCreatedEventDto.java:12]: and here

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package com.kamyczki.commons.stone.events;
+
+import lombok.*;
+
```
Comment (Saszq) [inline server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java:4]: pyk

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+@Data
+@Getter
+@AllArgsConstructor
+@RequiredArgsConstructor
+@ToString
+public class StoneEventDto {
+    protected String id;
+    protected String stoneId;
+
```
Comment (Saszq) [inline server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java:15]: bum

File: server/other/kafka/docker-compose.yml
Comment (Saszq) [inline server/other/kafka/docker-compose.yml:1]: Delete this file

Comment (mex1can0) [reply server/other/kafka/docker-compose.yml:1]: delete this from internet!

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/StoneKafkaConstants.java (hunk 2 above)
Comment (Isaribi) [reply server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/StoneKafkaConstants.java:7]: Intended :) but can be removed with only one field

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java (hunk 4 above)
Comment (Isaribi) [reply server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java:4]: This one is auto formatted

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java (hunk 5 above)
Comment (Isaribi) [reply server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneEventDto.java:15]: Intended but sure

File: server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneCreatedEventDto.java (hunk 3 above)
Comment (Isaribi) [reply server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/events/StoneCreatedEventDto.java:12]: Kay

File: server/other/kafka/docker-compose.yml
Comment (Isaribi) [reply server/other/kafka/docker-compose.yml:1]: Whoops, I forgot I put this here :)

**Pull request merged**